README.md

```diff
@@ -16,19 +16,19 @@ Repository to describe, develop, document and test the Release Management proces
 
 ## Release Process Information
 
-**The first public release of the Release Management documents with version v0.2.0 is availalbe in [r1.2](https://github.com/camaraproject/ReleaseManagement/releases/tag/r1.2)**
+**The second public release of the Release Management documents with version v0.3.0 is availalbe in [r2.1](https://github.com/camaraproject/ReleaseManagement/releases/tag/r2.1)**
```

Comment:
r2.1 is not a public release but a pre-release and the version number is v0.3.0-rc.1, not v0.3.0.

Reply:
I removed all version or release related information from the README and added a local reference to the CHANGELOG.md.